docker/oss-fuzz-builder/checkout_commit.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google LLC
```

Comment:
brought this up [before](https://github.com/google/fuzzbench/pull/306#discussion_r445719758) but not sure if there was a conclusion: have you considered using explicit `git checkout <commit>` commands in the Dockerfile-s instead of using this script?

Reply:
Yes, the same has a TODO after your suggestion. Afaik, it can be done but still requires more work to figure out the name for the directory where the checkout needs to be done, specifically in case of multiple clone commands and when the name of the directory is not same as that of the repository (as opposed to finding it using a loop currently).

Reply:
Update: The suggestion is to have the integrator must manually edit the Dockerfile to checkout the required commit as opposed to this being done by the script. This would be relatively cleaner. New TODO reflects the same.